### ISSUE TITLE
Could not pass event UpdateEvent to Thirst v1.4.1

### ISSUE THREAD

**Issue by FCRev** [GamerKing195/Thirst#1]
Looks like this happens every time a player logs off the server.

Could not pass event UpdateEvent to Thirst v1.4.1
org.bukkit.event.EventException
    at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:308) ~[spigot.jar:git-Spigot-455072c-43ab266]
    at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot.jar:git-Spigot-455072c-43ab266]
    at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502) [spigot.jar:git-Spigot-455072c-43ab266]
    at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487) [spigot.jar:git-Spigot-455072c-43ab266]
    at me.gamerzking.core.updater.Updater.run(Updater.java:27) [Thirst-1.4.1.jar:?]
    at org.bukkit.craftbukkit.v1_10_R1.scheduler.CraftTask.run(CraftTask.java:71) [spigot.jar:git-Spigot-455072c-43ab266]
    at org.bukkit.craftbukkit.v1_10_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:350) [spigot.jar:git-Spigot-455072c-43ab266]
    at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:723) [spigot.jar:git-Spigot-455072c-43ab266]
    at net.minecraft.server.v1_10_R1.DedicatedServer.D(DedicatedServer.java:400) [spigot.jar:git-Spigot-455072c-43ab266]
    at net.minecraft.server.v1_10_R1.MinecraftServer.C(MinecraftServer.java:668) [spigot.jar:git-Spigot-455072c-43ab266]
    at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:567) [spigot.jar:git-Spigot-455072c-43ab266]
    at java.lang.Thread.run(Thread.java:745) [?:1.8.0_91]
Caused by: java.lang.NullPointerException

It's also spamming the console every second, as soon as someone gets off and continues until I restart.
Any way I can fix this or is this a simple issue with the plugin?

**Comment by cadenkriese** [comment]
Sorry, think that's an issue on my end. I'll get a patch out of with the more thirst display configuration as soon as I get back from Spain(Sunday).

**Comment by cadenkriese** [comment]
Also thank you for making an issue on github.

**Comment by cadenkriese** [comment]
Alright so it was an easy quick fix so I wrote the code directly into GitHub, problem is I don't have a way to compile it since I'm not at home w/ my IDE (The program I normally use to code). Sooo the best I got is sharing this w/ my friend and having him compile it, that may or may not work and it will probably take a couple days, it might not work at all. In other words, I've written the code to fix the bug, however I can't put it on Spigot yet, going to try and have my friend do it, which may or may not work, I should have a fix out by Tuesday/Wednesday next week at the latest, tomorrow at the earliest.
